extensions/cornerstone/src/services/ViewportService/CornerstoneCacheService.ts

```diff
@@ -224,4 +228,12 @@ class CornerstoneCacheService {
 }
 
 const CacheService = new CornerstoneCacheService();
+
+export const registerCacheService = {
```

Comment:
This addresses your comment about having the registration information in the Service file itself.